app/models/forms/bulk_upload_lettings_resume/fix_choice.rb

```diff
@@ -56,7 +56,7 @@ def save!
       end
 
       def preflight_valid?
-        bulk_upload.choice != "create-fix-inline" && bulk_upload.choice != "bulk-confirm-soft-validations"
+        bulk_upload.choice.blank?
```

Comment:
This should basically prevent users from accessing the fix choice page if they made any kind of choice (either manually or by user being moved). I think it makes sense to discredit as many bulk uploads so that they don't linger with a possibility of the user stumbling upon the link and accidentally creating them. 

We might also separately want to update routing here if a log has setup errors, cause I don't think anything is stopping us from navigating to fix-choice page directly (?)
On top of that any bulk uploads for closed collection years should probably also not be accessible, or at least not uploadable